test/jwt.test.js

```diff
@@ -2390,12 +2389,12 @@ test('custom response messages', function (t) {
           method: 'get',
           url: '/verify',
           headers: {
-            authorization: 'Invalid Format'
+            authorization: 'Another Format'
```

Comment:
Keep it as `Invalid` as fine.

```suggestion
            authorization: 'Invalid Format'
```